### ISSUE TITLE
Running SummitTool throws "%1 is not a valid Win32 application" error

### ISSUE THREAD

**Issue by MichaelPStimpson** [google/summit-ast#44]
## Expected Behavior
Running `bazel run :SummitTool './Lead_Utility.cls` will output some basic information and provide an in-memory AST.

## Actual Behavior

Running the command produces the following error:
`
FATAL: ExecuteProgram(C:\users\michael\_bazel_michael\k757sjdo\execroot\__main__\bazel-out\x64_windows-fastbuild\bin\src\main\java\com\google\summit\SummitTool) failed: ERROR: src/main/native/windows/process.cc(202): CreateProcessW("C:\users\michael\_bazel_michael\k757sjdo\execroot\__main__\bazel-out\x64_windows-fastbuild\bin\src\main\java\com\google\summit\SummitTool"  C:\Users\Michael\Desktop\repos\summit-ast\Lead_Utility.cls): %1 is not a valid Win32 application.
 (error: 193)
`

## Steps to Reproduce the Problem

1. run `git clone https://github.com/google/summit-ast.git`
1. run `bazel build`
1. run `bazel run :SummitTool './Lead_Utility.cls'`

I've tried several variations for the path with forward/backward slashes, with/without single quotes, and the full path.

## Specifications

- OS: Windows 10 Home 10.0.19045 (64 bit)

**Comment by aaronpaulhurst** [comment]
Hi Michael,

Thanks for flagging this! We only aimed at Linux support for the first releases.  That said, I don’t see any reason — beyond some minor issues — why Windows should be a problem.  Let me set up a dev env and see if this can be fixed quickly.

**Comment by aaronhurst-google** [comment]
I spent a few hours working to build on a GCP windows VM.  I ran into quite a number of issues other than yours.

TLDR my conclusion was that bazel (and the common rules libraries that this depends on) are insufficiently stable to support development on Windows.  I don't plan to pursue this further.  (But this need not be the final word: it'd be great if someone is motivated to take this on!)  NOTE: continuing to support Windows users (e.g. who download a library from maven central) remains important.
